course_catalog/migrations/0046_add_learning_resource_offeror.py

```diff
@@ -22,6 +22,7 @@ def populate_offerors(apps, schema_editor):
     offered_by_mapping = {
         o.value: LearningResourceOfferor.objects.get_or_create(name=o.value)[0]
         for o in OfferedBy
+        if len(o.value) <= 32
```

Comment:
Yes, otherwise github CI tests will fail.  This is because the max character length for the field during this migration is 32 characters, and one of the new offered by values is longer than that (the max length was changed to 256 in a later migration).